instance/models/openedx_instance.py

```diff
@@ -134,7 +134,7 @@ def database_name(self):
         # for different services (e.g. xqueue) we don't want to use the maximum length here.
         allowed = string.ascii_letters + string.digits + '_'
         escaped = ''.join(char for char in name if char in allowed)
-        return truncate_name(escaped, length=50)
+        return truncate_name(escaped, length=40)
```

Comment:
Hmm... interesting, I didn't know we use `truncate_name` here. If we do this, do we even need to limit the subdomain name to 22 characters? Is there any other place where the length of the subdomain can be problematic, other than the database name?

Reply:
Yes, Sounds fair. I have removed the limitation on the sub-domain name.